src/ruleLoader.ts

```diff
@@ -29,12 +29,12 @@ export interface IEnableDisablePosition {
 
 export function loadRules(ruleConfiguration: {[name: string]: any},
                           enableDisableRuleMap: {[rulename: string]: Lint.IEnableDisablePosition[]},
-                          rulesDirectory?: string): Lint.IRule[] {
+                          rulesDirectories?: string | Array<string>): Lint.IRule[] {
```

Comment:
As I said above, I'm trying to keep backwards compatibility, so, it's still `string | string[]`, but I changed the syntax in commit https://github.com/palantir/tslint/commit/ee60a0cd1b858bd9555f21e9b4efdb501ebb570d.